### PR TITLE
Awesome Books with ES6

### PR DESCRIPTION
**Pull request by Aduda-Boaz**
This PR is for the Awesome books with ES6. It is a solo project for the Week 2 JavaScript module.
The project is for learning purposes on how to use ES6 syntax.
I used:
- HTML5
- CSS3
- JavaScript
- Luxon
The project is complete and ready for review.

### CODE REVIEW

File: modules/dist/localstorage.dev.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  value: true
+});
+exports.getFromLocalStorage = exports.addToLocalStorage = void 0;
+
+var _index = require("./index.js");
+
+//  eslint-disable-next-line import/no-cycle
+var addToLocalStorage = function addToLocalStorage(books) {
+  window.localStorage.setItem('books', JSON.stringify(books));
+};
```
Comment (mikethreels) [inline modules/dist/localstorage.dev.js]: - [x] linters help keep your code clean and optimize it, please don't disable eslint.
- [x] according to the requirements you should only use ES6 syntax, which includes arrow functions over the use of the function keyword, please adjust this.

File: modules/dist/localstorage.dev.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+exports.addToLocalStorage = addToLocalStorage;
+
+var getFromLocalStorage = function getFromLocalStorage() {
+  if (localStorage.length !== 0) {
+    var booksFromLocStg = JSON.parse(window.localStorage.getItem('books'));
+    booksFromLocStg.bookList.forEach(function (book) {
+      _index.books.bookList.push(book);
```
Comment (mikethreels) [inline modules/dist/localstorage.dev.js]: - [x] according to the requirements you should only use ES6 syntax, which includes arrow functions over the use of the function keyword, please adjust this.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Boaz, 

Good job on changing most of the functions to arrow functions 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._
- [x] In the index.js file on line 47 there is still a function using the function keyword, according to the requirements in this milestone you should use the ES6 arrow functions instead, please change this before requesting another review request.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Aduda-Boaz** [comment]
Thank you for your feedback:
Changes are made as suggested

**Review by mikethreels — APPROVED** [review]
Hi Boaz, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 :
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**